app/views/episodes/_form_tags.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="col-12 mb-4">
   <div class="card shadow border-0">
-    <div class="card-header card-header-secondary d-flex justify-content-between">
+    <div class="card-header card-header-light d-flex justify-content-between">
```

Comment:
On my (admittedly not-great) external monitor, I can barely distinguish this from the background.  Not a huge deal - but I suspect for accessiblility we might need to darken the `$light` a bit.